.github/workflows/benchmark.yml

```diff
@@ -25,23 +25,9 @@ jobs:
 
       - run: npm install -g npm@latest
 
-      - name: restore lerna
-        id: cache
-        uses: actions/cache@v3
-        with:
-          path: |
-            node_modules
-            package-lock.json
-            packages/*/node_modules
-            packages/*/package-lock.json
-            experimental/packages/*/node_modules
-            experimental/packages/*/package-lock.json
-          key: node-tests-${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('package.json', 'packages/*/package.json', 'experimental/packages/*/package.json') }}-04292022
-
       - name: Bootstrap
         run: |
-          npm install --ignore-scripts
-          npx lerna bootstrap --no-ci --hoist --nohoist='zone.js' --ignore @opentelemetry/selenium-tests
+          npm ci
```

Comment:
This doesn't need the `|` anymore because it isn't multiline

`run: npm ci` should be good enough